plugins/git/git.plugin.zsh

```diff
@@ -263,6 +262,9 @@ alias gtl='gtl(){ git tag --sort=-v:refname -n -l "${1}*" }; noglob gtl'
 
 alias gunignore='git update-index --no-assume-unchanged'
 alias gunwip='git log -n 1 | grep -q -c "\-\-wip\-\-" && git reset HEAD~1'
+
+alias gu='git pull'
+alias guum='git pull upstream master'
```

Comment:
With the changes ohmyzsh has made with to detecting main / master branch of a repo, I believe this alias will need to also be able to pull from ether. See this https://github.com/ohmyzsh/ohmyzsh/pull/9049